NuRadioReco/modules/io/RNO_G/readRNOGDataMattak.py

```diff
@@ -774,7 +809,7 @@ def get_event_by_index(self, event_index):
         Returns
         -------
 
-        evt: NuRadioReco.framework.event
+        evt: `NuRadioReco.framework.event.Event`
```

Comment:
out of curiosity, why?

Reply:
The former renders as text, the latter (online) as a clickable reference to the class.